jetstream_pt/engine.py

```diff
@@ -507,8 +508,13 @@ def _load_from_safetensors(self, path):
         arr = jax.device_put(f.get_tensor(key), self.sharding_by_name(key))
         assert tuple(model_weights.shape) == tuple(arr.shape), f"key: {key} error: {model_weights.shape} != {arr.shape}"
         weights[key] = arr
+        print(f"key:{key}, value shape:{arr.dtype}")
     freqs_cis = torch_xla2.tensor.t2j(self.pt_model.freqs_cis)
     weights['freqs_cis'] = jax.device_put(freqs_cis, self.replicated)
+
+    # if not self.env.bf16_enable:
```

Comment:
?

Reply:
Thanks for caching up. There is another approach to do float32 weights: convert them to float32 in loading time with bfloat16 checking points. In the end, I chose current approach use float32 checking points and comment the convert logics. 

Removed.